src/structures/paging/mapper/mod.rs

```diff
@@ -383,6 +383,7 @@ pub trait Mapper<S: PageSize> {
 /// changed the mapping of a page to ensure that the TLB flush is not forgotten.
 #[derive(Debug)]
 #[must_use = "Page Table changes must be flushed or ignored."]
+#[cfg_attr(not(feature = "instructions"), allow(dead_code))] // FIXME
```

Comment:
What is this FIXME about? The `allow(dead_code)`?

Reply:
Same as the other one: https://github.com/rust-osdev/x86_64/pull/476#discussion_r1551992988